src/img/icons/update_needed.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="200.000000pt" height="200.000000pt" viewBox="0 0 200.000000 200.000000"
+ preserveAspectRatio="xMidYMid meet">
+
+<g transform="translate(0.000000,200.000000) scale(0.100000,-0.100000)"
+fill="#000000" stroke="none">
+<path d="M750 1570 l0 -240 -75 0 c-41 0 -75 -3 -75 -6 0 -10 234 -323 241
+-324 4 0 61 73 128 163 l122 162 -75 3 -76 3 0 239 0 240 -95 0 -95 0 0 -240z"/>
+<path d="M743 869 c-66 -19 -86 -103 -37 -153 25 -25 32 -26 128 -26 108 0
+130 -7 124 -36 -3 -18 -16 -19 -180 -22 l-177 -2 -175 70 c-97 38 -190 70
+-207 70 -75 0 -118 -106 -67 -164 13 -14 160 -105 328 -202 363 -210 371 -214
+407 -214 26 0 557 95 596 106 16 5 17 25 17 242 l0 237 -105 53 -105 52 -258
+-1 c-141 0 -271 -5 -289 -10z"/>
+<path d="M1560 530 l0 -281 158 3 157 3 3 278 2 277 -160 0 -160 0 0 -280z
+m206 144 c22 -33 10 -72 -28 -93 -25 -14 -29 -14 -55 5 -34 25 -40 57 -19 90
+23 35 79 34 102 -2z"/>
+</g>
+</svg>
```

Comment:
Not certain this is the best icon for this purpose, perhaps we should just do a normal "download icon" (for example: https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTYw9r13IAz1pCtYjgF1Bh7To_87tzyEzIK0A&usqp=CAU) or maybe the spinner with an "!"?

Reply:
I used the static download icon